FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+ActionHandling.swift

```diff
@@ -48,9 +48,12 @@ extension ComposeViewController {
                 }
 
                 let contact = Recipient(recipient: recipient)
-                let contactWithFetchedKeys = try await pubLookup.fetchRemoteUpdateLocal(with: contact)
+                try await pubLookup.fetchRemoteUpdateLocal(with: contact)
+
+                if let updatedContact = try await localContactsProvider.searchRecipient(with: recipient.email) {
+                    handleEvaluation(for: updatedContact)
+                }
```

Comment:
Looks good, just worried that there will be some inconsistent ui state if it comes back empty. Empty result should still be handled to display that the recipients has no pubkeys, I think?

Reply:
Not sure what you mean. it won't return empty result as we fetch remote data above that line.

Reply:
See below comment